js/main.js

```diff
@@ -1 +1,24 @@
-// Your functions or code that should load on all pages goes here.
+function onVersionChange(form, animation) {
+    if (typeof version_prefixes == 'undefined' || typeof fields_per_version == 'undefined'
+        || !version_prefixes || !fields_per_version) {
+        return;
+    }
+    $.each(version_prefixes, function(version, prefix) {
+        let checkbox = form.find('#id_c_versions [value="' + version + '"]');
```

Comment:
Each checkbox looks like `<input type="checkbox" value="JP" ... />` and is placed inside a `<ul id="id_c_versions">...</ul>`, so we're using the name of the version to select it!